build.rs

```diff
@@ -20,7 +58,7 @@ fn main() {
     Build::new()
 		.cpp(true)
 		// https://github.com/facebook/yoga/blob/c5f826de8306e5fbe5963f944c75add827e096c3/BUCK#L13
```

Comment:
We can probably delete this comment now, or update it to this URL:

https://github.com/facebook/yoga/blob/9591210a7ac5289fa00f15d5068383612d3225b9/cmake/project-defaults.cmake#L6

Reply:
Yes, good point. I've just put a link to that whole file.